pmoves/tools/tz-cli/tz_cli/cli.py

```
@@ -0,0 +1,741 @@
+
+    click.echo()
+    click.echo("Configuration History:")
+    click.echo("=" * 60)
+
+    for backup in backups:
+        # Extract timestamp from filename
+        timestamp_str = backup.name.replace("tensorzero.toml.", "")
+        try:
+            timestamp = datetime.strptime(timestamp_str, "%Y%m%d_%H%M%S")
+            click.echo(f"  📅 {timestamp.strftime('%Y-%m-%d %H:%M:%S')} - {backup.name}")
+        except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)